spyder/widgets/shortcutssummary.py

```diff
@@ -106,6 +107,11 @@ def __init__(self, parent=None):
                 # Widgets
                 label_name = QLabel(name.capitalize().replace('_', ' '))
                 label_name.setFont(font_names)
+
+                if sys.platform.startswith('darwin'):
```

Comment:
If possible, I think it would be better to let Qt do the translation for us instead of doing it ourselves. So, instead of

```
if sys.platform.startswith('darwin'):
    keystr = keystr.replace('Ctrl', 
                                        'Cmd').replace('Alt', 'Option')
```

 could you please test if something like this would work on MacOS to solve this issue :

`keystr  = QKeySequence(keystr).toString(QKeySequence.NativeText)`

You'll need to add this import:

`from qtpy.QtGui import QKeySequence`

Reply:
You could also eliminate the platform check since that would be handled automatically.

Reply:
hehe looking at her profile, I think she's more than capable to figure the details by herself :P

Reply:
Hi! I am new to this and I followed what was on the issue. I'll change this as you suggested :)